docs/generator.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Bootstrap Framework
+sidebar_title: Bootstrap Framework
+description: How to quickly init testing environment
+---
+
+### Init Testing Environment
+
+Framework provides handy utility to bootstrap your testing environment with a single command:
+
+```shell
+npx flow-js-testing init
+```
+
+When you run this command in the terminal it will initiate npm package in your current directory and create `package.json` file.
+Then it will pull and install necessary dependencies. After the installation process is complete utility will create required
+config files for Babel, Jest and Flow CLI.
```

Comment:
```suggestion
When you run this command in the terminal it will initiate npm package in your current directory and create `package.json` file.
Then it will install dependencies. After the installation is finished, the utility will create the required
config files for Babel, Jest and Flow CLI.
```